site/src/api/api.ts

```diff
@@ -2274,6 +2275,19 @@ class ApiMethods {
 			throw new MissingBuildParameters(missingParameters, activeVersionId);
 		}
 
+		// Stop the workspace if it is already running.
```

Comment:
hmm. I'm a bit worried about adding this directly inside the `API` method. most everything in here is 1:1 with a backend endpoint, occasionally with minimal logic to convert from an options object to a query string or whatever. but this is actual application logic leaking in here.